### ISSUE TITLE
Getting exception after update to ^1

### ISSUE THREAD

**Issue by vladyslavstartsev** [antonioribeiro/google2fa-laravel#54]
```
> Illuminate\Foundation\ComposerScripts::postUpdate
> php artisan ide-helper:generate
Exception: You need to install the imagick extension to use this back end
Skipping \PragmaRX\Google2FALaravel\Facade.
Exception: You need to install the imagick extension to use this back end
Skipping \PragmaRX\Google2FALaravel\Facade.
```

and after that when I call `\PragmaRX\Google2FALaravel\Facade::isActivated()` for example I also get `BaconQrCode/Exception/RuntimeException with message 'You need to install the imagick extension to use this back end'`

Can you make that  `imagick` extension is not required, since, for example, I don't need to draw QR code, I just pass data to frontend and then it draws on frontend.

**Comment by antonioribeiro** [comment]
Yeah, this is unfortunate I agree, I will try to take a look at it. Thanks for reporting.

**Comment by aFluxx** [comment]
Hello,

This might be related to the QR package, As you can see here:
https://github.com/Bacon/BaconQrCode/issues/60

There's some more information in that issue.

**Comment by kamui545** [comment]
Using the SVG backend got rid of the exception for me (even though im generating QR Code via Google).

`'qrcode_image_backend' => \PragmaRX\Google2FALaravel\Support\Constants::QRCODE_IMAGE_BACKEND_SVG`
